wallet/src/Pos/Wallet/Web/Server/Handlers.hs

```diff
@@ -14,9 +14,8 @@ import           Ntp.Client (NtpStatus)
 
 import           Pos.Wallet.Web.Swagger.Spec (swaggerSpecForWalletApi)
 import           Servant.API ((:<|>) ((:<|>)))
-import           Servant.Generic (AsServerT, GenericProduct, ToServant,
```

Comment:
compilation of this file really took a long time! (some minutes)
will this change bring this down to a few seconds?
would be interesting to time it..
(@parsonsmatt suggested that Servant.Generic is probably the critical piece)

Reply:
Myabe, maybe not. I didn't benchmark it. I've just tried hard to ensure it build with LTS-12